RubberduckTests/Inspections/IllegalAnnotationsInspectionTests.cs

```diff
@@ -164,51 +164,6 @@ End Sub
             }
         }
 
-        [Test]
-        [Category("Inspections")]
-        public void SingleFolderAnnotation_NoResult()
-        {
-            const string inputCode =
-                @"'@Folder ""Foo""
-Public Sub Foo()
-    Const const1 As Integer = 9
-End Sub";
-
-            var vbe = MockVbeBuilder.BuildFromSingleStandardModule(inputCode, out _);
-            using (var state = MockParser.CreateAndParse(vbe.Object))
-            {
-
-                var inspection = new IllegalAnnotationInspection(state);
-                var inspector = InspectionsHelper.GetInspector(inspection);
-                var inspectionResults = inspector.FindIssuesAsync(state, CancellationToken.None).Result;
-
-                Assert.IsFalse(inspectionResults.Any());
-            }
-        }
```

Comment:
not sure why this test was deleted

Reply:
This test was checking whether an annotation is not duplicated, which is now the responsibility of the new inspection.